SwiftGit2/OID.swift

```diff
@@ -68,7 +68,7 @@ extension OID: Hashable {
 			self.oid.id.6,
 			self.oid.id.7
 		]
-		return reduce(bytes, 0, { (hash, byte) in
+		return bytes.reduce(0, combine:{ (hash, byte) in
```

Comment:
Style-wise, this would be better with trailing closure syntax:

```swift
return bytes.reduce(0) { (hash, byte) in
	return Int(hash << 8) | Int(byte)
}
```

Would you mind adjusting it?